### PR TITLE
feat: bootstrap default values

### PR DESCRIPTION
**Pull request by CasLubbers**
Default values are stored in different places (projects). We want to adhere to single source of truth design pattern and store all values in Gitea.

### CODE REVIEW

File: values-schema.yaml [hunk 1]
```diff
@@ -2114,7 +2120,7 @@ properties:
           apiServer:
             additionalProperties: false
             properties:
-              enabled:
+              create:
```
Comment (j-zimnowoda) [inline values-schema.yaml:2123]: is create the right property?

Comment (CasLubbers) [reply values-schema.yaml:2140]: It was the one currently used in the defaults. So it was either change the value in the defaults.yaml to enabled or change the openapi schema.

**Comment by j-zimnowoda** [comment]
I performed some tests with bootstrapping the values with 
```
npm run bootstrap-dev
# and 
npm run bootstrap-dev-with-repo
```
contributed one fix and now it is looking good.

File: values-schema.yaml (hunk 1 above)
Comment (j-zimnowoda) [reply values-schema.yaml:2140]: I check it is a correct change. I also added the values migration for the schema change

**Review by j-zimnowoda — APPROVED** [review]

**Review by j-zimnowoda — COMMENTED** [review]
IMHO we should not store letsencryptCA and letsencryptRootCA in the values.
Perhaps moving them to derived.gotmpl is a good way to go.

**Pull request merged**